js/apps/account-ui/src/account-security/SigningIn.tsx

```diff
@@ -113,115 +113,110 @@ const SigningIn = () => {
+            </SplitItem>
+            {container.createAction && (
+              <SplitItem isFilled>
+                <div className="pf-u-float-right">
+                  <MobileLink
+                    onClick={() =>
+                      login({
+                        action: container.createAction,
+                      })
+                    }
+                    title={t("setUpNew", [
+                      t(container.displayName as TFuncKey),
+                    ])}
+                  />
+                </div>
               </SplitItem>
-              {container.createAction && (
-                <SplitItem isFilled>
-                  <div className="pf-u-float-right">
-                    <MobileLink
-                      onClick={() =>
-                        login({
-                          action: container.createAction,
-                        })
-                      }
-                      title={t("setUpNew", [
-                        t(container.displayName as TFuncKey),
-                      ])}
-                    />
-                  </div>
-                </SplitItem>
-              )}
-            </Split>
+            )}
+          </Split>
 
-            <DataList aria-label="credential list" className="pf-u-mb-xl">
-              {container.userCredentialMetadatas.length === 0 && (
-                <EmptyRow
-                  message={t("notSetUp", [
-                    t(container.displayName as TFuncKey),
-                  ])}
-                />
-              )}
+          <DataList aria-label="credential list" className="pf-u-mb-xl">
+            {container.userCredentialMetadatas.length === 0 && (
+              <EmptyRow
+                message={t("notSetUp", [t(container.displayName as TFuncKey)])}
+              />
+            )}
 
-              {container.userCredentialMetadatas.map((meta) => (
-                <DataListItem key={meta.credential.id}>
-                  <DataListItemRow>
-                    <DataListItemCells
-                      className="pf-u-py-0"
-                      dataListCells={[
-                        ...credentialRowCells(meta),
-                        <DataListAction
-                          key="action"
-                          id={`action-${meta.credential.id}`}
-                          aria-label={t("updateCredAriaLabel")}
-                          aria-labelledby={`cred-${meta.credential.id}`}
-                        >
-                          {container.removeable ? (
-                            <ContinueCancelModal
-                              buttonTitle="remove"
-                              buttonVariant="danger"
-                              modalTitle={t("removeCred", [
-                                label(meta.credential),
-                              ])}
-                              modalMessage={t("stopUsingCred", [
-                                label(meta.credential),
-                              ])}
```

Comment:
This still needs to be restored.

Reply:
It's likely because this comes from `ui-shared` which needs to be rebuilt. If you remove this code and then re-buid the project from scratch I think you will not get this message.